sonar-project.properties

```
@@ -17,6 +17,7 @@ sonar.links.homepage=https://github.com/Apodini/Apodini
 # Path is relative to the sonar-project.properties file.
 sonar.sources=Sources
 sonar.exclusions=Sources/Apodini/Components/ComponentBuilder.swift, \
+                 Sources/Apodini/RelationshipIdentificationBuilder.swift, \
```

I think you used the wrong file path

```suggestion
                 Sources/Apodini/Relationships/RelationshipIdentificationBuilder.swift, \
```

---

Good caught. Resolved via 6c2b9bf